_posts/blog/2019-08-13-Ang-Li-Three-People-in-Turing.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Three amazing people in Turing"
+categories: blog
+date: 2019-08-13
+modified:
+share: true
+author: Ang Li
+---
+
+Some amazing people I met in Turing. :)
+
+## Bo Wang
+
+Bo Wang is a visiting researcher at Turing and work with Professor Terry Lyons based at Oxford. 
```

Comment:
Bo Wang is a visiting researcher at the Turing and works with Professor Terry Lyons based at Oxford.